packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -167,6 +167,30 @@ abstract class WebViewPlatformController {
     throw UnimplementedError(
         "WebView getTitle is not implemented on the current platform");
   }
+
+  /// Sets the initial scale of page content.
+  ///
+  /// Only supported on Android.
+  ///
+  /// [scaleInPercent] must be between 0 and 100 inclusive.
+  ///
+  /// The behavior for the default initial scale depends on whether the WebView
+  /// supports the "viewport" HTML meta tag, whether the WebView is set to use a
+  /// wide viewport, and whether the WebView loads pages in overview mode, that
+  /// is, zooms out the content to fit on screen by width.
+  ///
+  /// If the content fits into the WebView control by width, then the zoom is
```

Comment:
I'm not sure I understand this.
Does it mean to say that setting the initial scale to 100% will make the page fit the webview's width?

Reply:
I think it depends on the webpage and what the settings for 
https://developer.android.com/reference/android/webkit/WebSettings.html#getUseWideViewPort()
https://developer.android.com/reference/android/webkit/WebSettings.html#getLoadWithOverviewMode() are.

For example if you set it to 20:
<img width="225" alt="Screen Shot 2020-02-18 at 4 52 54 PM" src="https://user-images.githubusercontent.com/10687576/74791502-6bb39480-526f-11ea-9c81-5ecd84a052c0.png">

If you set it to 100:
<img width="225" alt="Screen Shot 2020-02-18 at 4 54 55 PM" src="https://user-images.githubusercontent.com/10687576/74791516-74a46600-526f-11ea-833b-7b69f29d470c.png">

Reply:
Please explain this in the doc comment